### PR TITLE
rewrote crud-mutations page and added PostgraphileRC to usage-cli

### PR DESCRIPTION
**Pull request by rsubr**
- updated crud-mutations page to follow the blog example from the previous section. Old incomplete superheroes examples have been removed.
- updated usage-cli page with PostGraphile RC usage.

### CODE REVIEW

File: src/pages/postgraphile/crud-mutations.md [hunk 1]
```diff
@@ -11,19 +11,60 @@ via the `--disable-default-mutations` CLI setting (or the
...
+`primary key` column.
 
 You also get the following query fields:
 
-- superhero - Returns a single `Superhero` using its globally unique `ID`.
-- superheroByRowId - Returns a single `Superhero` using its database row ID.
-- allSuperheroes - Returns a [connection](/postgraphile/connection/) enabling pagination through all (visible) superheroes.
+- person - Returns a single `Person` using its globally unique `ID`.
+- personById - Reads a single `Person` using its globally unique `ID`.
+- personByEmail - Reads a single `Person` using its globally unique `ID`.
```
Comment (benjie) [inline src/pages/postgraphile/crud-mutations.md]: Typo; you meant email

File: src/pages/postgraphile/usage-cli.md [hunk 2]
```diff
@@ -16,128 +16,147 @@ npm install -g postgraphile
...
+
 ### CLI options
 
 There are more CLI options available to customise the GraphQL server:
 
 <!-- CLI_DOCBLOCK_BEGIN -->
 
-* `-h`, `--help`  
+* `-h`, `--help`
   output usage information
```
Comment (benjie) [inline src/pages/postgraphile/usage-cli.md]: Please restore these double spaces; they cause markdown to force a line break, I found this was the best way of formatting this page.

**Comment by benjie** [comment]
Hey @rsubr, would you like to make these changes or should I?

**Comment by rsubr** [comment]
@benjie the double space removal was due to some VS Code setting. Sorry. I've been in the hospital for the past few days and unable to contribute further for another few weeks. Please go ahead and make the changes.

**Comment by benjie** [comment]
Sorry to hear that; get well soon!

**Pull request merged**